src/libraries/System.Data.OleDb/src/OleDbException.cs

```diff
@@ -70,13 +70,8 @@ internal static OleDbException CreateException(UnsafeNativeMethods.IErrorInfo er
             string? message = null;
             string? source = null;
             OleDbHResult hr = 0;
-
-            if (null != errorInfo)
```

Comment:
Why don't we need a null check here?

Reply:
There discussion about that https://github.com/dotnet/runtime/pull/54822#discussion_r660048158